### PR TITLE
Adding a more human readable format for last login

### PR DESCRIPTION
**Pull request by Microwave22**
I saw the note about making the last_login a more human readable format so I did it.

### CODE REVIEW

File: dnx_frontend/dfe_dnx_main.py [hunk 1]
```diff
@@ -495,7 +495,7 @@ def dnx_login():
         authenticated, username, user_role = Authentication.user_login(request.form, request.remote_addr)
         if (authenticated):
             session['username'] = username
-            # session['expire_time'] = time.time() + 30
+            # session['expire_time'] = + 30
```
Comment (Microwave22) [inline dnx_frontend/dfe_dnx_main.py:498]: Don't know remember doing this but it shouldn't affect anything.

File: dnx_frontend/dfe_dnx_main.py [hunk 2]
```diff
@@ -606,12 +606,13 @@ def update_session_tracker(username, user_role=None, remote_addr=None, *, action
 
     with ConfigurationManager('session_tracker', file_path='dnx_frontend') as session_tracker:
         stored_tracker = session_tracker.load_configuration()
-
+        
         if (action is CFG.ADD):
+            last_login = datetime.datetime.now()
```
Comment (Microwave22) [inline dnx_frontend/dfe_dnx_main.py:611]: I found that putting datetime into a variable helps. Don't know why just what I have seen.

File: dnx_frontend/dfe_dnx_main.py [hunk 3]
```diff
@@ -606,12 +606,13 @@ def update_session_tracker(username, user_role=None, remote_addr=None, *, action
...
             stored_tracker['active_users'][username] = {
                 'user_role': user_role,
                 'remote_addr': remote_addr,
-                'logged_in': time.time(), # NOTE: can probably make this human readable format here.
+                'logged_in': last_login,
```
Comment (Microwave22) [inline dnx_frontend/dfe_dnx_main.py:615]: Replacing the old confusing time with a new human readable one.

**Review by DOWRIGHTTV — CHANGES_REQUESTED** [review]
thanks for taking time to put a PR in! in this case, the human readable needs to be string format/ serializable since the "stored_tracker" object gets serialized into json and written to disk. if i remember correctly datetime has methods to convert to string somewhat easily with a formatting string.

**Pull request closed without merging**